src/rulesets/Base/broadcast.jl

```diff
@@ -128,13 +131,14 @@ function split_bc_pullbacks(cfg::RCR, f::F, args::Vararg{Any,N}) where {F,N}
         rrule_via_ad(cfg, f, a...)
     end
     function back_generic(dys)
-        deltas = unzip_broadcast(backs, unthunk(dys)) do back, dy  # (could be map, sizes match)
+        deltas = unzip_broadcast(backs, dys) do back, dy  # (could be map, sizes match)
             map(unthunk, back(dy))
         end
         dargs = map(unbroadcast, args, Base.tail(deltas))
         df = ProjectTo(f)(sum(first(deltas)))
         return (NoTangent(), NoTangent(), df, dargs...)
     end
+    back_generic(dys::AbstractThunk) = back_generic(unthunk(dys))
     back_generic(z::AbstractZero) = (TRI_NO..., map(Returns(z), args)...)
     return ys3, back_generic
```

Comment:
Somehow Diffractor fed this pullback a thunk containing a ZeroTangent. And... well maybe I tried this with #671 which changes this to `unzip_map`, and that was not happy about the zero. So I changed them all to have 3 methods, not two.

Reply:
Now some removed, as this caused inference problems.